### PR TITLE
Always send ctrl

### PR DESCRIPTION
**Pull request by jbarber**
This patch makes it possible to send Ctrl keys to the server, which
makes it possible to do Ctrl-click to, for example, do multiselect on
the web.

We do this by always sending ctrl. This then breaks pasting text
(Ctrl-v), becaue the first character of the pasted text is assumed by
the server to be part of the ctrl sequence. We "fix" this, by sending a
ctrl key-up just before every character in the pasted sequence.

### CODE REVIEW

**Comment by jbarber** [comment]
@ukd1 / @gallois can you give this the once over? Tested against IOS in xcode (for paste bug) + Chrome on Win7.

File: Flashlight-VNC/src/com/flashlight/vnc/VNCClient.as [hunk 1]
```diff
@@ -570,28 +572,20 @@ package com.flashlight.vnc
...
-						}
-					}
 				}
 				
 				//logger.info("keysym "+keysym);
 				
 				rfbWriter.writeKeyEvent(event.type == flash.events.KeyboardEvent.KEY_DOWN,keysym);
+
+				// prevent onTextInput() from firing
+				//event.stopImmediatePropagation();
```
Comment (gallois) [inline Flashlight-VNC/src/com/flashlight/vnc/VNCClient.as]: why is this commented out?

Comment (jbarber) [inline Flashlight-VNC/src/com/flashlight/vnc/VNCClient.as]: It's a note to future-me to remind me that you can stop events propagating, I cargo culted it from this related [PR](https://github.com/rainforestapp/Flashlight-vnc/pull/9/files#diff-5e601c60176c463882f8ef03ad5d3f5cL542).

Enabling/disabling it doesn't seem to do anything though... So the comment is wrong.

Comment (jbarber) [inline Flashlight-VNC/src/com/flashlight/vnc/VNCClient.as]: The obvious (:troll:) reason for it not doing anything is that there are no other handlers for `KeyboardEvent.KEY_UP` or `KEY_DOWN` events, so `onLocalKeyboardEvent` is the final handler anyway.

**Comment by gallois** [comment]
all right :) lgtm

**Pull request merged**